app/client/src/features/accounts/UserSideNav.tsx

```diff
@@ -126,14 +141,38 @@ export function UserSideNav({ queryRef, onClick }: UserSideNavProps) {
                     </StyledListItemIcon>
                     <ListItemText primary='User Guide' />
                 </StyledListItem>
-                <StyledSubheader>Organizations</StyledSubheader>
-                <StyledDivider />
-                <StyledListItem onClick={handleClick('My Organizations')} selected={selected === 'My Organizations'}>
-                    <StyledListItemIcon>
-                        <ListIcon />
-                    </StyledListItemIcon>
-                    <ListItemText primary='My Organizations' />
-                </StyledListItem>
+                <RoleGuard organizer={data.isOrganizer}>
+                    <>
+                        <StyledSubheader>Organizations</StyledSubheader>
+                        <StyledDivider />
+
+                        <StyledListItem
+                            onClick={handleClick('My Organizations')}
+                            selected={selected === 'My Organizations'}
+                        >
+                            <StyledListItemIcon>
+                                <ListIcon />
+                            </StyledListItemIcon>
+                            <ListItemText primary='My Organizations' />
+                        </StyledListItem>
+                    </>
+                </RoleGuard>
```

Comment:
In the current implementation of the `RoleGuard` component, I think this would redirect if the user is not an organizer, which I'm not sure is the behavior we want.  Is that correct?

Reply:
yea that was very unfinished code from what I remember, but I did push more changes that should make more sense now!